packages/core/src/config/minimal.ts

```diff
@@ -110,7 +110,17 @@ const minimal: PluginStyleguideConfig<'built-in'> = {
   },
   arazzoRules: {
     spec: 'error',
-    'parameters-no-body-inside-in': 'off',
+    'parameters-not-in-body': 'off',
+    'source-description-type': 'off',
+    'version-enum': 'warn',
```

Comment:
It's related to Spot. I'd make it 'off' in `minimal`.

Reply:
There was a comment from @lornajane mentioning that it can be `warn`.
I like warn more - as it gives some additional context for users.

Reply:
I'm fine with warn, just wanted to be sure it's ok.